spec/models/version_spec.rb

```diff
@@ -3,9 +3,9 @@
 require "spec_helper"
 require "support/shared_examples/queries"
 require "support/shared_examples/active_record_encryption"
-
+require "support/custom_object_changes_adapter"
```

Comment:
Very small nit: probably add a blank line to separate the `require`s from the `module` declaration (as before)?